client/src/Reserve/DualTimeSelector.js

```diff
@@ -1,11 +1,15 @@
 import React, { useState, useEffect } from 'react';
 
-function DualTimeSelector() {
+function DualTimeSelector({ timeList, setTimeList }) {
   const [startTime, setStartTime] = useState('09:00');
   const [endTime, setEndTime] = useState('10:00');
-  const [timeList, setTimeList] = useState([]);
+  // const [timeList, setTimeList] = useState([]);
   const [name, setName] = useState(`VUL-0`);
 
+  useEffect(() => {
```

Comment:
This new useEffect looks like it'd be an infinite loop. Does it cause that recursive re-render problem?